app/models/post.rb

```diff
@@ -10,4 +10,27 @@ def comments_counter
   def likes_counter
     likes.count
   end
+
+  def update_posts_counter
+    author.update(posts_count: author.posts_count + 1)
+  end
+
+  def update_likes_counter
+    likes_count = likes.count
+    update(likes_count:)
+  end
+
+  def update_comments_counter
+    comments_count = comments.count
+    update(comments_count:)
+  end
```

Comment:
- [ ] Please implement the following method in the `Comment` model as the method is used whenever a `Comment` is created not a `Post`, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=a%20given%20post.-,Comments,A%20method%20that%20updates%20the%20comments%20counter%20for%20a%20post.,-Likes) requirement.